ci-operator/step-registry/redhat-chaos/krkn-hub-tests/redhat-chaos-krkn-hub-tests-chain.yaml

```diff
@@ -11,6 +11,7 @@ chain:
   - name: PROFILE_TYPE
     default: "reporting"
   steps:
+  - ref: redhat-chaos-network-chaos
```

Comment:
lets move this test case till after the cluster-density-v2 so we have load on the cluster when performing this scenario

Reply:
Sure!